CHANGELOG.md

```diff
@@ -5,18 +5,18 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
-### Updated
-*
-
-### Changed
-*
 
 ## [1.2.0] - 2023-11-17
 
-### Updated
-* Updated Practitioner resource type to include gmp numbers.
-* Updated the way Observation comments are mapped so that blood pressure NarrativeStatements are not duplicated into
-  in a separate observation.
+### Added
+
+* Populate `identifier` field for `Practitioner` resources using the General Practitioner (GMP) number if available.
+
+### Fixed
+
+* Prevent `NarrativeStatement / text` sat within a blood pressure `CompoundStatement` from being mapped
```

Comment:
`Updated` is not a [valid type of changelog entry](https://keepachangelog.com/en/1.1.0/#how). #pedant.

But being less pendantic, updated is a very generic term, and could cover all and every type of change made. `Fixed` feels more appropriate given it was a bug, most users won't notice a difference, and no additional functionality has been added/removed that the NME needs to be aware of.